src/migration/MigrationExecutor.ts

```diff
@@ -90,18 +90,17 @@ export class MigrationExecutor {
     }
 
     /**
-     * Returns an array of all migrations.
-     */
-    public async getAllMigrations(): Promise<Migration[]> {
```

Comment:
Imo we shouldn't remove this as it's a breaking change, perhaps worth deprecating and removing in v1, wdyt?

Reply:
Sure. Done.